comms.py

```diff
@@ -0,0 +1,62 @@
+from mcu import *
+from controls import *
+
+class Communications:
+    def __init__(self, mcuVAR: MCUInterface, controller: controls):
+        self.mcuVAR = mcuVAR
+        self.controller = controls
+
+        MOTOR_CLAW = 0 
+        MOTOR_ONE = 1
+        MOTOR_TWO = 2
+        MOTOR_THREE = 3 
+
+    def read_send(key_pressed):
+        initial_calibration_percent = 100
```

Comment:
Add typing for `initial_calibration_percent` and `value_multiplier`.

Reply:
not really necessary, since they are really just constants. Move them to the file scope and define them as such: `INITIAL_CALIBRATION_PERCENT = 100`